management/src/main/java/org/thethingsnetwork/management/async/AsyncDiscovery.java

```diff
@@ -69,7 +69,6 @@ private AsyncDiscovery(DiscoveryGrpc.DiscoveryFutureStub _stub) {
                     try {
                         ManagedChannel ch = ManagedChannelBuilder
                                 .forAddress(_host, _port)
-                                .usePlaintext(true)
```

Comment:
I'm afraid we need this for private networks still

Reply:
But the current SDK fails because of trying to connect with the secured server. Is there any workaround?

Reply:
Can we make it an option?

Reply:
Optionally added the secure connection.